src/app/loaders/express/cors.ts

```diff
@@ -0,0 +1,19 @@
+import cors from 'cors'
+
+import { app, corsWhiteList } from '../../config/config'
+
+export const corsMiddleware = () => {
+  return cors({
+    origin: function (origin, callback) {
+      // If no origin or from self, disable CORS
+      if (!origin || app.appUrl === origin) {
+        callback(null, false)
```

Comment:
sorry I may be misunderstanding this but shouldn't this be `true`? if it's a server to server request or the request originates from our own domain, we should allow it and set the ACAO header to reflect the request origin?

Reply:
also just double checking, does this allow incoming requests from AWS for email notifications?

Reply:
disabling cors will just result in status quo, no?

good point about the AWS email notifications, can we check that easily?

Reply:
>disabling cors will just result in status quo, no?

yes for same-origin requests, but I'm not sure about server-to-server requests (eg email notifications). can check on staging, just submit an email mode form and see if the email notification gets logged

Reply:
i assume aws will send without origin since it's not a browser and should not be affected

Reply:
i'll test on staging

Reply:
works on staging without issues